WordPress/Classes/WordPress.xcdatamodeld/WordPress 94.xcdatamodel/contents

```diff
@@ -0,0 +1,843 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="15400" systemVersion="18G95" minimumToolsVersion="Xcode 7.3" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
+    <entity name="AbstractPost" representedClassName="AbstractPost" isAbstract="YES" parentEntity="BasePost">
+        <attribute name="autosaveContent" optional="YES" attributeType="String" syncable="YES"/>
+        <attribute name="autosaveExcerpt" optional="YES" attributeType="String" syncable="YES"/>
+        <attribute name="autosaveIdentifier" optional="YES" attributeType="Integer 64" defaultValueString="0" usesScalarValueType="NO" syncable="YES"/>
```

Comment:
Oops, I didn't notice that. I don't recall adding a default value - I'll do a test to see if Xcode is adding it, but either way I think this can be removed through the Xcode's interface.
Thanks for spotting this.

Reply:
Removed now. A default value of zero is Xcode's default for certain attribute types (not sure if this is normal or Xcode's default behavior changed).

<img width="300" alt="Screen Shot 2019-12-01 at 10 18 36 PM" src="https://user-images.githubusercontent.com/1898325/69924431-1fd02b00-148a-11ea-8901-03e171088e61.png">